src/views/CardStack.js

```diff
@@ -81,7 +81,8 @@ const RESPOND_THRESHOLD = 12;
 /**
  * The distance of touch start from the edge of the screen where the gesture will be recognized
  */
-const GESTURE_RESPONSE_DISTANCE = 35;
+const GESTURE_RESPONSE_DISTANCE_SIDE = 35;
```

Comment:
nit: can we name it as `GESTURE_RESPONSE_DISTANCE_HORIZONTAL` and `GESTURE_RESPONSE_DISTANCE_VERTICAL`?